Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/PressableButtonInspector.cs

```diff
@@ -116,10 +108,16 @@ private ButtonInfo GatherCurrentInfo()
         {
             ButtonInfo info = new ButtonInfo();
 
-            info.TouchCageLocalBounds = new Bounds(touchCage.center, touchCage.size);
+            info.LocalCenter = touchable.LocalCenter;
+            info.PlaneExtents = touchable.Bounds;
 
-            Vector3 pressDirLocal = (touchable != null) ? -1.0f * touchable.LocalForward : Vector3.forward;
-            Vector3 upDirLocal = (touchable != null) ? touchable.LocalUp : Vector3.up;
+            Vector3 pressDirLocal = (touchable != null) ? touchable.LocalPressDirection : Vector3.forward;
+            Vector3 upDirLocal = Vector3.up;
+
+            if (touchable is NearInteractionTouchable touchableConcrete)
```

Comment:
Woah cool, I didn't know you could do that!

Reply:
Yes, it's available as of C# 7.  This actually reminds me...
@julenka, does MRTK target .NET 4.7 and above?

If you target .NET 4.6 or below, then I should remove the use of this syntax.

Reply:
Regarding PressableButtonsTestCanvas.prefab, it is an example prefab.  I named it that because it lives right next to the existing "TestCanvas.prefab" which is all of the pre-existing UnityUI stuff in the HandInteractionExamples scene.  I was trying to be consistent with the existing naming scheme.

[Hand tracking prefabs](https://github.com/AdamMitchell-ms/MixedRealityToolkit-Unity/tree/unityui-nearfield-improvements2/Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Prefabs)